gen/MotorControllerGroup.yml

```diff
@@ -16,12 +16,18 @@ classes:
             if (args.size() < 1) {
               throw py::value_error("requires 1+ arguments");
             }
-            using Vector = std::vector<std::shared_ptr<MotorController>>;
-            Vector v = args.cast<Vector>();
-            for (auto iter = v.cbegin(); iter != v.cend(); ++iter) {
-              if (!*iter) {
-                throw py::value_error("Arguments must not be None");
+            std::vector<std::shared_ptr<MotorController>> v;
+            v.reserve(args.size());
+            int i = 1;
```

Comment:
Should this count from 0 instead?

Reply:
I feel like most people count arguments from 1? Particularly when speaking. "first parameter, second parameter..."

Reply:
Fair enough. I wonder whether any stdlib functions have a similar error message.